mcodingbot/plugins/info.py

```diff
@@ -20,6 +20,7 @@ async def links(ctx: Context) -> None:
         title="Links",
         color=CONFIG.theme,
         description=(
+            f"**[mCoding site]({CONFIG.mcoding_site})**\n"
```

Comment:
Maybe?
```suggestion
            f"**[mCoding website]({CONFIG.mcoding_site})**\n"
```

Reply:
Also rename `mcoding_site` to `mcoding_website` for consistency